lib/src/aes_gcmsiv.c

```diff
@@ -50,7 +50,7 @@ struct key_context {
 struct aes {
     int has_hw;
     union {
-        struct mbedtls_aes_context mbedtls;
+        struct generic_aes_context mbedtls;
```

Comment:
What about also renaming this field to `generic`? To match with the field name in `struct polyval` and to give more insight of the associated source file?